modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -279,6 +323,72 @@ class TestFunctional
         }
     }
 
+    template <typename T>
+    inline void initPointRandU(cv::RNG& rng, T& pt)
+    { ::initPointRandU(rng, pt); }
+
+    // initialize std::vector<cv::Point_<T>>/std::vector<cv::Point3_<T>>
+    template <typename T, template <typename> class Pt>
+    void initPointsVectorRandU(const int sz_in, std::vector<Pt<T>> &vec_)
+    {
+        cv::RNG& rng = theRNG();
+
+        vec_.clear();
+        vec_.reserve(sz_in);
+
+        for (int i = 0; i < sz_in; i++)
```

Comment:
There is a warning here:
```
gapi_tests_common.hpp(339): warning C4702: unreachable code
```

Reply:
Could you try to print the values here (`sz_in`, `vec_`). There might me an uninitialized value somewhere

Reply:
@alalek please, could you help with these two problems?

Reply:
As it is a test code (not a public header), so you can eliminate this warning through `#pragma` for MSVS 2015 (use `_MSC_VER < 1910/*MSVS 2017*/`)